composer.json

```diff
@@ -16,7 +16,7 @@
     "ext-libxml": "*",
     "ext-zlib": "*",
     "incenteev/composer-parameter-handler": "^2.2",
-    "openconext/monitor-bundle": "^4.1",
+    "openconext/monitor-bundle": "^4.3.0",
```

Comment:
May I suggest `^4.3`, for no functional added beneficial reason. Just for looks. So feel free to leave as is!